ethcore/verification/src/verification.rs

```diff
@@ -187,18 +187,19 @@ fn verify_uncles(block: &PreverifiedBlock, bc: &dyn BlockProvider, engine: &dyn
 			// 5
 			// 6											7
 			//												(8 Invalid)
-
-			let depth = if header.number() > uncle.number() { header.number() - uncle.number() } else { 0 };
-			if depth > MAX_UNCLE_AGE as u64 {
-				return Err(From::from(BlockError::UncleTooOld(OutOfBounds {
-					min: Some(header.number() - depth),
+			let depth = if header.number() > uncle.number() {
+				header.number() - uncle.number()
+			} else {
```

Comment:
This branch is hit for both equality and less-than (`header.number() == uncle.number()` and `header.number() < uncle.number()`); are both cases "is-brother" errors? Sounds like if the uncle height is greater than the block height it should be `OutOfBounds`? (But I'm not sure I understand the logic here).

Reply:
The behaviour here hasn't changed from the original commit, but you're right. This should be probably `OutOfBounds` error